### DISCUSSION TITLE
Idea: Page for links to other resources

### DISCUSSION THREAD

**Discussion by alansill** [hpc-social/hpc-social.github.io#43] [Ideas]
As a Project on the web site, it would be good to add links for resources offered by others that are of interest to the HPC community. We could have for example "Blogs", "Podcasts" (already suggested by @vsoch under https://github.com/hpc-social/hpc-social.github.io/issues/40), and "Other Groups" as categories, either as separate projects (see for example PR https://github.com/hpc-social/hpc-social.github.io/pull/41 for "Podcasts" that may get merged while we discuss this item) or as one Project page with multiple tabs.

Possible ideas for Blogs to include: 
 - https://newsletter.researchcomputingteams.org
 - https://bssw.io/blog_posts

Possible links to other groups:
- https://casc.org
- https://carcc.org
- ...

**Comment by vsoch** [comment]
I mentioned this somewhere else, but it would be cool to have a community syndicated blog! It would live in a blog repository or similar. I actually don’t know (or haven’t looked) at which community members have blogs - but if they do it would be fun to provide.

**Reply by alansill** [reply]
Okay, so that's cool, so now you've done it and people are already adding pull requests as suggested for other blogs to add at https://github.com/hpc-social/blog/ to appear in published form at https://hpc.social/blog as described at https://hpc.social/projects/blog/!

For our next act: Community Groups? Or Conferences and Events? (With tools for community input on each, of course!)

**Reply by vsoch** [reply]
The groups could go on the map, right? And a list of conferences/  events would be useful, although I don't have any good ideas for cool interfaces beyond a list (that we manually update?) Maybe throw the question out to the community and see what feedback we get.

**Reply by alansill** [reply]
Hmm, many groups don't have a definite location, or are national or even global in location. Examples from the list above, or even hpc.social itself, Open GridForum, etc., which strive to be global. Also haveing a blurb about eachorg, maybe under a reveal triangle, would be helpful in deciding whether to visit the link.

**Reply by vsoch** [reply]
oh gotcha, we need sort of a registry of groups? Somewhere to explore? A gallery (with images and metadata popups) would be appropriate.

**Reply by alansill** [reply]
This need seems to be also in the process of being addressed at [https://cnct.ci/resources](https://cnct.ci/resources)

Maybe we should approach them for some cross-linking of our organizations?

**Reply by alansill** [reply]
Looking at [https://cnct.ci](https://cnct.ci) in more detail, it seems to aim at being very general but in practice the list of resources it is aiming at serving seems very aimed at those related tot he ACCESS-CI project only. There are a few groups aimed at more general discussion, but the list is primarily just a list and not a platform itself. Still might be useful to link from our own list if we make one.

**Reply by alansill** [reply]
I put in a request to have hpc.social linked as a card on the cnct.ci site, but several days have gone by and it isn't there yet, so it looks like they are keeping the focus on ACCESS-related projects. 

We could make our own equivalent of which this and other orgs like CASC, CARCC, and the RSE orgs could be among the listings.

**Reply by vsoch** [reply]
Sure! What should we call it - an organization page?

**Reply by alansill** [reply]
How about "Organizations" with an interface like the others that lets people request to be added, possibly wiht an upload for a graphic element (with guidelines) and possibly a description field. We can look at the cnct.ci input form as an example.

**Reply by vsoch** [reply]
okay I can't promise I can work on this soon - there are too many other things I want/need to do before it, but it will eventually come up in the Q.

**Comment by vsoch** [comment]
@alansill are we missing another resource type? We currently have chat, podcasts, community blogs, jobs, and blogs.

**Reply by alansill** [reply]
For the last category mentioned I meant external resources like other related organizations etc.

**Reply by vsoch** [reply]
Gotcha, so what did you have in mind for that?
